<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerDialog</class>
 <widget class="QDialog" name="SellerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Створити Seller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>&lt;h3&gt;Новий Seller&lt;/h3&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_id"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_lastName">
       <property name="text">
        <string>Прізвище:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_lastName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_firstName">
       <property name="text">
        <string>Ім'я:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_firstName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_patronymic">
       <property name="text">
        <string>По батькові:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_patronymic"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_address">
       <property name="text">
        <string>Адреса:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_address"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_account">
       <property name="text">
        <string>Номер рахунку:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_accountNumber"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_products">
     <property name="text">
      <string>Товари:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_products"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_addProduct">
     <property name="text">
      <string>Додати товар</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_create">
     <property name="text">
      <string>Створити</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
